data/test/scenarios/test_role_types.cfg

```diff
@@ -41,6 +53,17 @@
     [/event]
 )}
 
+#####
+# API(s) being tested: [role],[have_unit]role=
+##
+# Actions:
+# Spawn some units on the map.
+# Assign those unit types to a role, choosing a unit based on the unit type.
```

Comment:
"Assign a role to a unit" rather than "assign unit types to a role". A role is a just a property of a unit, roles aren't objects that can have lists of unit types assigned to them.